test/integration/default/controls/archive_spec.rb

```diff
@@ -39,7 +47,7 @@
     it { should exist }
     its('group') { should eq 'prometheus' }
   end
-  describe file('/usr/lib/systemd/system/prometheus.service') do
+  describe file("#{service_dir}/prometheus.service") do
```

Comment:
This is weird that tests were passing before this change

Reply:
Debian has a `/usr/lib/systemd/` but I find no example of having `'/usr/lib/systemd/system/` under it. Perhaps it does work, but the hosts I checked do not use it? I'm not sure, honestly.

Reply:
Also, before YOU added the tests for services in #43 , services were not tested, so perhaps the files were added there but the service was not using THOSE files but the package-provided ones?

Reply:
> This is weird that tests were passing before this change

Actually it was failling... I missed it : https://travis-ci.com/github/saltstack-formulas/prometheus-formula/jobs/377094793#L4182